docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -283,6 +284,61 @@ tasks.withType(KotlinCompile)
 </tab>
 </tabs>
 
+## Kotlin compiler fallback strategy
+
+The Kotlin compiler's fallback strategy is to run a compilation outside a Kotlin daemon if the daemon somehow fails. 
+If the Gradle daemon is on, the compiler uses the ["In process" strategy](#defining-kotlin-compiler-execution-strategy). 
+If the Gradle daemon is off, the compiler uses the "Out of process" strategy.
+
+When this fallback happens, you have the following warning lines in your Gradle's build output:
+
+```
+Could not perform incremental compilation: Could not connect to Kotlin compile daemon
+Could not connect to kotlin daemon. Using fallback strategy.
```

Comment:
Since 1.8.0 the message is slightly changed:
```suggestion
Failed to compile with Kotlin daemon: java.lang.RuntimeException: Could not connect to Kotlin compile daemon
[exception stacktrace]
Using fallback strategy: Compile without Kotlin daemon
Try ./gradlew --stop if this issue persists.
```

Reply:
In the case of disabled fallback strategy the message would look like:
```
Failed to compile with Kotlin daemon. Fallback strategy (compiling without Kotlin daemon) is turned off. Try ./gradlew --stop if this issue persists.
```

Reply:
Thanks! Seems like this message 
>In the case of disabled fallback strategy the message would look like:

is not needed in docs, right?

Reply:
I think yes, it's not necessary here